src/java.desktop/unix/native/libwlsplashscreen/splashscreen_sys.c

```diff
@@ -361,15 +364,19 @@ SplashInitPlatform(Splash *splash) {
     splash->wl_state->wl_shm = NULL;
     splash->wl_state->wl_compositor = NULL;
     splash->wl_state->wl_subcompositor = NULL;
+    splash->wl_state->wl_output = NULL;
```

Comment:
Since there are so many ` = NULL` lines, it now seems easier to just do `calloc()` instead.

Reply:
Sure, should splash->buffers and splash->main_buffer also be `calloc()`ed?

Reply:
Perhaps in the future; for now, let's keep the patch to the necessary minimum.